docs/iviewer_viewing.rst

```diff
@@ -78,25 +80,41 @@ Step-by-Step
 
    |image0b|
 
+#. Make a maximum intensity Projection of a z-stack by clicking the ``stack`` icon |image0c| in the bottom-left corner of the central pane, just under the z-scrollbar.
+
+#. Note that on the vertical z-scrollbar, there are now two sliders. Adjust the sliders to select the desired portion of the z-stack to be projected. The central pane will show a preview of your Projection.
+
+#. Save the Projection by selecting ``File > Save Projection as new image``. In the following dialog, click either ``Navigate to Image in Webclient`` or ``Open Image in ivewer``.
+
 #. Save the viewport by selecting in the top-left corner ``File > Save Viewport as PNG``.
 
 #. Compare the two channels of one of the Images in multi-image view:
 
-    - Double-click on the thumbnail of the already opened Image in the left-hand pane of the viewer. This will open the Image again in a new window.
+   - Double-click on the thumbnail of the already opened Image in the left-hand pane of the viewer. This will open the Image again in a new window.
 
-    - Select one of the windows and in the right-hand pane switch the green channel off.
+   - Select one of the windows and in the right-hand pane switch the green channel off.
 
-    - On the other window, switch the blue channel off.
+   - On the other window, switch the blue channel off.
 
        .. image:: images/Iviewer12.png
 
-#. We can synchronize the viewers by adding them both to the same ``sync Group``.
+#. Work with images in the multi-view mode. There are several ways how to get full views of the images from the thumbnails in the left-hand column.
+
+   - ``Clicking once`` on a thumbnail opens the corresponding image in the currently active central pane viewer window, replacing the image in that viewer window.
+
+   - ``Double-clicking`` on a thumbnail opens a new central pane viewer window with the double-clicked thumbnail image.
+
+   - ``Dragging and dropping`` a thumbnail into an existing central pane viewer window will replace the image in that window with the image corresponding to the thumbnail which was just dropped into that window.
+
+   - ``Dragging  and dropping`` a thumbnail onto the white canvas area in central pane will create a new viewer window.
```

Comment:
extra space

Reply:
fixed in https://github.com/ome/omero-guide-iviewer/pull/8/commits/e78101cfbbef53f7a3376254d5ef3f2622df758b and also removed some spaces in https://github.com/ome/omero-guide-iviewer/pull/8/commits/9bdb66940865a1865f953359228fd72be72dcbca